README.md

```diff
@@ -124,6 +124,7 @@ Example:
       writable-database-white-list:
       - my_writable_db1
       - my_writable_db2
+      - user_db_*
```

Comment:
We should document this better, probably add a section in the whitelisting describing that you can either do full db names or use reg ex and then link to the Java Pattern documentation https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html